services-custom/dynamodb-enhanced/src/test/java/software/amazon/awssdk/extensions/dynamodb/mappingclient/functionaltests/AsyncBasicCrudTest.java

```diff
@@ -376,11 +381,17 @@ public void putWithConditionThatFails() {
                    .join();
     }
 
+    @Test
+    public void deleteItemInterfaceToOperationSucceeds() {
+        mappedTable.deleteItem(r -> r.key(Key.create(stringValue("id-value"), stringValue("sort-value")))).join();
+        mappedTable.deleteItem(DeleteItemEnhancedRequest.builder()
+                                                        .key(Key.create(stringValue("id-value"), stringValue("sort-value")))
+                                                        .build()).join();
+    }
+
```

Comment:
I think we should be able to make basic assertions here beyond 'it didn't blow up'.